SPECS-EXTENDED/perl-Test-RandomResult/perl-Test-RandomResult.spec

```diff
@@ -0,0 +1,96 @@
+Name:           perl-Test-RandomResult
+Version:        0.001
+Release:        14%{?dist}
+Summary:        Test that results of a running code look random
+License:        GPL-1.0-or-later OR Artistic-1.0-Perl
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://metacpan.org/release/Test-RandomResult/
+Source0:        https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Test-RandomResult-%{version}.tar.gz
+BuildArch:      noarch
+BuildRequires:  make
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 6.76
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+# Run-time
+BuildRequires:  perl(Exporter) >= 5.57
+BuildRequires:  perl(Test2::API)
+# Tests
+BuildRequires:  perl(blib)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(IO::Handle)
+BuildRequires:  perl(IPC::Open3)
+BuildRequires:  perl(Test::More)
+Requires:       perl(Exporter) >= 5.57
+
+%global __requires_exclude %{?__requires_exclude:%__requires_exclude|}perl\\(Exporter\\)\\s*$
+
+%description
+Test that results of a running code look random
+
+%prep
+%setup -q -n Test-RandomResult-%{version}
+
+%build
+perl Makefile.PL INSTALLDIRS=vendor NO_PACKLIST=1 NO_PERLLOCAL=1
+%{make_build}
+
+%install
+%{make_install}
+%{_fixperms} $RPM_BUILD_ROOT/*
+
+%check
+make test
+
+
+%files
+%license LICENSE
+%doc Changes README
+%{perl_vendorlib}/*
+%{_mandir}/man3/*
+
+%changelog
+* Tue Dec 10 2024 Kevin Lockwood <v-klockwood@microsoft.com> - 0.001-14
+- Add as a dependency of perl-Color-RGB-Util
```

Comment:
Since we are importing from Fedora, could you please add the following to the changelog in addition to the above. 
- Initial Azure Linux import from Fedora 41 (license: MIT)